patches/third_party/ffmpeg/002-Fix-build_ffmpeg.py-and-automatic-gernerate-ffmpeg-m.patch

```diff
@@ -0,0 +1,7896 @@
+From 12c1704063ff6f2b1e599b58a7937f8c20a59264 Mon Sep 17 00:00:00 2001
+From: Wang Qing <wangqing-hf@loongson.cn>
+Date: Thu, 14 Sep 2017 19:07:25 +0800
+Subject: [PATCH] Fix build_ffmpeg.py and automatic gernerate ffmpeg mips64el
```

Comment:
For the `002-Fix-build_ffmpeg.py-and-automatic-gernerate-ffmpeg-m.patch`, it seems that most changes are actually automatically generated? I think we should auto-generate the files in the `update` script, otherwise this patch would be impossible to maintain when we do Chrome upgrades.

Reply:
ok